src/wp-admin/includes/image.php

```diff
@@ -1143,3 +1239,62 @@ function _copy_image_file( $attachment_id ) {
 
 	return $dst_file;
 }
+
+/**
+ * Returns an array with the list of valid mime types that a specific mime type should be converted into.
+ * For example an `image/jpeg` should be converted into an `image/jpeg` and `image/webp`. The first type
+ * is considered the primary output type for this image.
+ *
+ * @since 6.0.0
+ *
+ * @param $attachment_id int The attachment ID.
+ * @return array<string, array<string>> An array of valid mime types, where the key is the mime type and the value is the extension type.
```

Comment:
A the end of this line it may be helpful to say which the order of the array of strings represents. Like, is the first value the `to` or the `from`?

Reply:
also maybe we should define if this should be limited to two types in the array. (what happens when three are provided?)

Reply:
Good suggestion I will add some clarifying language here. 

We do allow for more than two types here, for example, you might want to generate JPEG, WebP and AVIF formats.

Reply:
added